src/main/kotlin/no/fg/hilflingbackend/controller/PhotoController.kt

```diff
@@ -75,6 +75,8 @@ class PhotoController(
     @RequestParam("securityLevelIdList") securityLevelIdList: List<UUID>,
     @RequestParam("gangIdList") gangIdList: List<UUID>,
     @RequestParam("photoGangBangerIdList") photoGangBangerIdList: List<UUID>,
+    @RequestParam("albumIdList") albumIdList: List<UUID>,
```

Comment:
Hadde det vært en ide å sende alt som er i listene i samme object/liste? Typ i stedet for å sende:
gangIdList: List, photoGangBangerIdList: List, ... etc
så kan vi sende (navngiving bør være bedre enn i mitt eksempel her:
`
paramList: [
  {
    gangId: UUID,
    photoGangBangerId: UUID,
    ...
  },
  {
    gangId: UUID,
    photoGangBangerId: UUID,
    ...
  },
  ...
]
`